Tensile/KernelWriterAssembly.py

```diff
@@ -13158,6 +13168,10 @@ def generateStoreInUnrollPostLoop(self, kernel, isOptNLL):
 
           for x in self.StoreCTemplate.items():
             kStrPL += str(x)
+          # StorSyncOpt
+          if kernel["StoreSyncOpt"]:
+            kStrPL += "s_sleep %d // optimization: sync and wait\n" %(kernel["StoreSyncOpt"]-1)
```

Comment:
Does this need both sleep and barrier?

Reply:
I think so.
I copied this code from other locations.